From e551d960442c903ce00cf5be88a8665cf59ce476 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@sourcemage.org>
Date: Sun, 9 Aug 2026 08:36:48 +0000
Subject: [PATCH] libbase: don't use the non-standard sys/cdefs.h

sys/cdefs.h is a glibc/bionic-internal header and does not exist on
musl libc, which broke the musl build with:

    vendor/libbase/include/android-base/test_utils.h:20:10: fatal error:
    sys/cdefs.h: No such file or directory

Upstream-Status: Pending
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 vendor/libbase/include/android-base/test_utils.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/vendor/libbase/include/android-base/test_utils.h b/vendor/libbase/include/android-base/test_utils.h
index e57138f..1e3cfc6 100644
--- a/vendor/libbase/include/android-base/test_utils.h
+++ b/vendor/libbase/include/android-base/test_utils.h
@@ -16,9 +16,6 @@
 
 #pragma once
 
-// For __BIONIC__
-#include <sys/cdefs.h>
-
 #if defined(__BIONIC__)
 #include <malloc.h>
 #include <stdio.h>
